config.js.template

```diff
@@ -26,9 +28,13 @@ module.exports = {
   hsmUsername: null,
   hsmPassword: null,
 
+  // Only change the URL if you want to use the Sandbox API under 'https://sandbox-api.fireblocks.io'
   fireblocksUrl: 'https://api.fireblocks.io',
   fireblocksApiKey: null,
+  // This should be a string containing the secret (with line breaks)
```

Comment:
Is it possible to use multiline strings here? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#multi-line_strings

Reply:
Yes, I personally used multiline string when developing and it worked without any issue.